geotrellis-python/geotrellis/python/edited_avro/avro_builder.py

```diff
@@ -0,0 +1,239 @@
+from __future__ import absolute_import
```

Comment:
Looks like it came from here, so I guess you mean an edited version of it https://github.com/Yelp/avro/blob/trunk/lang/py/src/avro/avro_builder.py

I see this file is not in trunk avro. Did they delete it in a later version? Is there another method of building up a schema that they recommend?